.github/workflows/nightly.yml

```diff
@@ -0,0 +1,89 @@
+            configure: cmake -B build -DWITH_FFMPEG_JOBS="$(nproc)"
+            release: |
+              cmake -B build -DCMAKE_BUILD_TYPE=Release -DWITH_FULL_RELEASE=Off -DWITH_NIGHTLY_RELEASE=On -DWITH_CLUB_FANTASTIC=Off
+              cmake --build build --target package
+            path: build/ITGmania-*-NIGHTLY-git-*-Linux-no-songs.tar.gz
+          - os: ubuntu-24.04-arm
+            name: Ubuntu ARM
+            arch: arm
+            install: sudo apt-get update && sudo apt-get install -y libasound2-dev libgl-dev libglu1-mesa-dev libgtk-3-dev libjack-dev libmad0-dev libpulse-dev libudev-dev libxinerama-dev libx11-dev libxrandr-dev libxtst-dev nasm
+            configure: cmake -B build -DWITH_FFMPEG_JOBS="$(nproc)" -DWITH_MINIMAID=OFF
+            release: |
+              cmake -B build -DCMAKE_BUILD_TYPE=Release -DWITH_FULL_RELEASE=Off -DWITH_NIGHTLY_RELEASE=On -DWITH_CLUB_FANTASTIC=Off
+              cmake --build build --target package
+            path: build/ITGmania-*-NIGHTLY-git-*-Linux-no-songs.tar.gz
+          - os: macos-latest
+            name: macOS (M1)
+            arch: arm64
+            install: brew install nasm
+            configure: cmake -B build -DCMAKE_OSX_ARCHITECTURES=arm64 -DWITH_FFMPEG_JOBS="$(sysctl -n hw.logicalcpu)"
+            release: |
+              cmake -B Build/release-arm64 -DCMAKE_BUILD_TYPE=Release -DWITH_FULL_RELEASE=Off -DWITH_NIGHTLY_RELEASE=On -DCMAKE_OSX_ARCHITECTURES=arm64 -DWITH_CLUB_FANTASTIC=Off
+              cmake --build Build/release-arm64 --target package
+            path: Build/release-arm64/ITGmania-*-NIGHTLY-git-*-macOS-M1-no-songs.dmg
+          - os: macos-13
+            name: macOS (Intel)
+            arch: x86_64
+            install: brew install nasm
+            configure: cmake -B build -DCMAKE_OSX_ARCHITECTURES=x86_64 -DWITH_FFMPEG_JOBS="$(sysctl -n hw.logicalcpu)"
+            release: |
+              cmake -B Build/release-x86_64 -DCMAKE_BUILD_TYPE=Release -DWITH_FULL_RELEASE=Off -DWITH_NIGHTLY_RELEASE=On -DCMAKE_OSX_ARCHITECTURES=x86_64 -DWITH_CLUB_FANTASTIC=Off
+              cmake --build Build/release-x86_64 --target package
+            path: Build/release-x86_64/ITGmania-*-NIGHTLY-git-*-macOS-Intel-no-songs.dmg
+          - os: windows-latest
+            name: Windows
+            arch: x64
+            install: |
+              $vsPath = &"C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe" -latest -products * -requires Microsoft.VisualStudio.Component.VC.Tools.x86.x64 -property installationPath
+              &"$vsPath\VC\Auxiliary\Build\vcvarsall.bat" x64
+            configure: cmake -B build -DWITH_FFMPEG_JOBS="$env:NUMBER_OF_PROCESSORS"
+            release: |
+              cmake -B Build\release -DWITH_FULL_RELEASE=Off -DWITH_NIGHTLY_RELEASE=On -DWITH_CLUB_FANTASTIC=Off
+              cmake --build Build\release --config Release --target package
+            path: Build/release/ITGmania-*-NIGHTLY-git-*-Windows-no-songs.exe
```

Comment:
The wildcard on the git SHA tends to match multiple files, is it built multiple times? e.g. there are three in https://github.com/itgmania/itgmania/actions/runs/14769996036/job/41468382313#step:7:9